tests/tests/kakarot_api.rs

```diff
@@ -126,3 +133,40 @@ async fn test_kakarot_get_starknet_transaction_hash_with_none_tx_hash(#[future]
 
     drop(server_handle);
 }
+
+#[rstest]
+#[awt]
+#[tokio::test(flavor = "multi_thread")]
+async fn test_kakarot_get_config(#[future] katana: Katana, _setup: ()) {
```

Comment:
can you add just a couple of comments in the test and line breaks so the code is a little more readable?